pkg/chunk/purger/purger.go

```diff
@@ -66,7 +66,7 @@ func newPurgerMetrics(r prometheus.Registerer) *purgerMetrics {
 	m.oldestPendingDeleteRequestAgeSeconds = promauto.With(r).NewGauge(prometheus.GaugeOpts{
 		Namespace: "cortex",
 		Name:      "purger_oldest_pending_delete_request_age_seconds",
-		Help:      "Age of oldest pending delete request in seconds",
+		Help:      "Age of oldest pending delete request in seconds, since they are over their cancellation period",
```

Comment:
I'm not sure I do understand what you mean by "since they are over their cancellation period". Could you clarify it?

Reply:
We do not process requests until they are over their cancellation period i.e they are not allowed to be cancelled anymore. I wanted to make it clear because people would think the age is since they are created which is not the case here.
I can remove it if it looks too confusing. Also, we have a metric to count the number of pending requests https://github.com/cortexproject/cortex/blob/35b756fa80c80210a678c59ff65a004d2bf0a018/pkg/chunk/purger/purger.go#L71 and here also pending requests are those which are over their cancellation period.
What do you think? I think we should either add that statement to both or none.

Reply:
Now I got it and I think having it in the description is fine. Agree that we should add the same to `pendingDeleteRequestsCount` too.